next.config.js

```diff
@@ -75,4 +80,6 @@ nextConfig.workerLoaderOptions = {
   name: 'static/[hash].worker.js',
 }
 
-module.exports = withTM(withWorkers(withImages(withSass(withCss(nextConfig)))))
+module.exports = withSourceMaps(
+  withTM(withWorkers(withImages(withSass(withCss(nextConfig)))))
+)
```

Comment:
For the future: it might be worth to combine functions in functional way to make things more clear.

```js
const combine = (...functions) => (...args) => functions.reduce(f => f(...args))
```

I am not sure my example is correct but you can test it out later.